juniper_official/chassis/check-fan-state-rpm.rule

```diff
@@ -165,10 +174,29 @@ healthbot {
                     }
                 }
                 /*
-                 * Sets color to green when RPM is less than 50%.
+                 * Sets color to yellow when RPM percent is below low threshold ($low-threshold)
+                 * and the rpm-percent value is anomalous.
+                 */ 
+                term is-rpm-percent-anomalous {
+                    when {
+                        less-than-or-equal-to "$rpm-percent" "$low-threshold";
+                        equal-to "$rpm-percent-anomaly" 1;
+                    }
+                    then {
+                        status {
+                            color yellow;
+                            message "$fan-name RPM percent($rpm-percent%) is not between the acceptable range of ($rpm-percent-anomaly-lower and $rpm-percent-anomaly-upper)";
```

Comment:
Seems like return of JAIML is upper-boundary and lower-boundary. Then field will be like <field>-upper-boundary instead for <field>-upper

Reply:
added the missed "-boundary" tag